Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp

```diff
@@ -106,16 +102,32 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian(
 
     for (MFIter mfi(*Mfield[0], TilingIfNotGPU()); mfi.isValid(); ++mfi) /* remember to FIX */
     {
-        auto& mag_alpha_mf = macroscopic_properties->getmag_alpha_mf();
-        auto& mag_gamma_mf = macroscopic_properties->getmag_gamma_mf();
-        auto& mag_exchange_mf = macroscopic_properties->getmag_exchange_mf();
-        auto& mag_anisotropy_mf = macroscopic_properties->getmag_anisotropy_mf();
+        auto& mag_alphax_mf = macroscopic_properties->getmag_alpha_mf(0);
+        auto& mag_alphay_mf = macroscopic_properties->getmag_alpha_mf(1);
+        auto& mag_alphaz_mf = macroscopic_properties->getmag_alpha_mf(2);
+        auto& mag_gammax_mf = macroscopic_properties->getmag_gamma_mf(0);
+        auto& mag_gammay_mf = macroscopic_properties->getmag_gamma_mf(1);
+        auto& mag_gammaz_mf = macroscopic_properties->getmag_gamma_mf(2);
+        auto& mag_exchangex_mf = macroscopic_properties->getmag_exchange_mf(0);
+        auto& mag_exchangey_mf = macroscopic_properties->getmag_exchange_mf(1);
+        auto& mag_exchangez_mf = macroscopic_properties->getmag_exchange_mf(2);
+        auto& mag_anisotropyx_mf = macroscopic_properties->getmag_anisotropy_mf(0);
+        auto& mag_anisotropyy_mf = macroscopic_properties->getmag_anisotropy_mf(1);
+        auto& mag_anisotropyz_mf = macroscopic_properties->getmag_anisotropy_mf(2);
```

Comment:
This is really minor - for clarity it would be better to call this mag_alpha_xface_mf 
alphax alphay alphaz could be confusing as they read like vector components of alpha.
Same for gamma, exchange, and anisotropy

Reply:
fixed in e100af929711a4a0acb52f075eab59d9bd608718